x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java

```diff
@@ -200,6 +214,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.field("hits", hits);
                 builder.field("misses", misses);
                 builder.field("evictions", evictions);
+                builder.field("hits_time_in_millis", hitsTimeInMillis);
+                builder.field("misses_time_in_millis", missesTimeInMillis);
```

Comment:
Random drive-by, but can we make these use `builder.humanReadableField("hits_time_in_millis", "hits_time", new TimeValue(hitsTimeInMillis))` so that the `?human` flag can be used for the stats? Sometimes it's nice to be able to see "oh 14.2 minutes of hit time" instead of doing the millisecond-to-human conversion.